pkg/commands/edit/fix/fix.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fix
+
+import (
+	"log"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/kustomize/pkg/commands/kustfile"
+	"sigs.k8s.io/kustomize/pkg/fs"
+)
+
+// NewCmdFix returns an instance of 'fix' subcommand.
+func NewCmdFix(fSys fs.FileSystem) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "fix",
```

Comment:
Should also modify the log messages at
https://github.com/kubernetes-sigs/kustomize/blob/master/pkg/types/kustomization.go#L157
to tell user to run this command.

Reply:
updated the log messages